`__wbindgen_thread_destroy` has optional params (#3703)
rustwasm/wasm-bindgen: CHANGELOG.md

* Add additional constructor to `DataView` for `SharedArrayBuffer`.
  [#3695](https://github.com/rustwasm/wasm-bindgen/pull/3695)

> * Remove experimental status from `wasm_bindgen::module()` and update docs to say it works with `--target web`.

Reviewer: ```suggestion

* Stabilize `wasm_bindgen::module()`.
```
I don't think we ever added entries for documentation changes, so let's not start it here.

  [#3690](https://github.com/rustwasm/wasm-bindgen/pull/3690)

### Fixed

* The DWARF section is now correctly modified instead of leaving it in a broken state.
  [#3483](https://github.com/rustwasm/wasm-bindgen/pull/3483)

> * Update the TypeScript signature of `__wbindgen_thread_destroy` to indicate that it has optional parameters.

Reviewer: ```suggestion

* Update the TypeScript signature of `__wbindgen_thread_destroy` to indicate that it's parameters are optional.
```
